modules/gateway-task/README.md

```diff
@@ -0,0 +1,7 @@
+# Gateway Task
+
+This module deploys a Consul gateway task as part of the service mesh.
+It is designed so you can easily modify existing [`ecs_task_definition`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition)
+resources to use this module instead.
```

Comment:
We should update this description. It could clarify which gateway types are currently supported.